### PR TITLE
seasonal new version blog post

### PR DESCRIPTION
**Pull request by christophsax**
need to update seasonalview on CRAN, but otherwise good from my side. How about you, @DivadNojnarg?

### CODE REVIEW

**Comment by christophsax** [comment]
The shinylive post is unhappy again: https://github.com/cynkra/cynkrablog/actions/runs/11440223400/job/31825596055?pr=64

Any idea, @DivadNojnarg? Thanks!

The second package is now on CRAN the post is ready, removing the wip.

**Comment by DivadNojnarg** [comment]
@christophsax. I suspect the issue could be coming from a recent change in quarto. In my wip PR, I fixed the version of quarto to avoid this kind of things: https://github.com/cynkra/cynkrablog/pull/63/commits/17f372edc281eb413d8f576aaaa6c95848f9a2ba#diff-551d1fcf87f78cc3bc18a7b332a4dc5d8773a512062df881c5aba28a6f5c48d7R20. I'd suggest to try the same in your branch to see.

File: posts/2024-10-21-seasonal-1.10/index.qmd [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+optionally containing some arguments. For
+example, in order to set the 'variables' argument of the 'regression' spec equal
+to `td` and `ao1999.jan`, the input to `seas` can be specified like this:
+
+```{r, eval = FALSE}
+m <- seas(AirPassengers, regression.variables = c("td", "ao1955.jan"))
+```
+
+*seasonal* has a flexible mechanism to read data from X-13ARIMA-SEATS. With the
+`series` function, it is possible to extract all output that can be
```
Comment (DivadNojnarg) [inline posts/2024-10-21-seasonal-1.10/index.qmd:60]: to extract all outputs? Missing _s_?

**Comment by christophsax** [comment]
@DivadNojnarg I tried, but I am getting down a rabbit hole again. Now I have to fix the links in your older blog post...

I guess I will wait for you to either you fix your old post in an isolated PR, or you publish your post first, and I will build on top.

**Comment by DivadNojnarg** [comment]
I also had some troubles with false positives with Lychee which I resolved with: https://github.com/cynkra/cynkrablog/pull/63/commits/17591c9749884776ae186d35cf99826bcc5f101c. Strava also got fixed here: https://github.com/cynkra/cynkrablog/pull/63/commits/a658ec16c314b87e45e674fdea573d9ae161c9f0 even though that's anotherLychee issue (the link is truly accessible). As ou said, I can try to publish my PR first and you build on top.

**Comment by christophsax** [comment]
closing, in favor of #65

**Pull request closed without merging**